src/vcpkg/vcpkgpaths.cpp

```diff
@@ -134,6 +134,18 @@ namespace vcpkg
         return parsed_config_opt;
     }
 
+    static std::vector<std::string> merge_overlays(const std::vector<std::string>& cli_overlays,
```

Comment:
```suggestion
    static std::vector<std::string> merge_strings(const std::vector<std::string>& cli_overlays,
```

It's also being used for triplets so this name isn't quite right.

Reply:
It merges overlay ports and overlay triplets though...

Reply:
Megaderp!